clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/input/recordreader/avro/IdentityVeniceRecordReader.java

```diff
@@ -37,6 +37,11 @@ public Object getAvroValue(ByteBuffer keyBytes, ByteBuffer valueBytes) {
     throw new VeniceUnsupportedOperationException("getAvroValue in IdentityVeniceRecordReader");
   }
 
+  @Override
+  public Long getRecordRMD(ByteBuffer inputKey, ByteBuffer inputValue) {
+    return -1L;
+  }
```

Comment:
I don't remember why we needed this class, but I'll take another look at this. I remember there was a good reason I did this, and this was important 😅